griptape/artifacts/image_artifact.py

```diff
@@ -0,0 +1,11 @@
+from typing import Optional
+from attr import define, field
+from griptape.artifacts import BlobArtifact
+
+
+@define(frozen=True)
+class ImageArtifact(BlobArtifact):
+    mime_type: str = field(default="image/png", kw_only=True)
```

Comment:
Not sure this should be the default. Can we make it more generic? Seems like we [might have options](https://superuser.com/questions/979135/is-there-a-generic-mime-type-for-all-image-files).

Reply:
Agree that a default doesn't quite make sense here. Updated to require the mime_type value.